Cargo.lock

```diff
@@ -6472,7 +6472,7 @@ dependencies = [
 
 [[package]]
 name = "moonbeam-cli"
-version = "0.34.0"
```

Comment:
We should also update the version in `Cargo.toml` for moonbeam-cli

Reply:
good catch!